src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -1028,7 +1028,11 @@ private static void VerifyModelForOutVarInNotExecutableCode(SemanticModel model,
 
             var typeInfo = model.GetTypeInfo(decl);
             Assert.Equal(expectedType, typeInfo.Type);
-            Assert.Equal(expectedType, typeInfo.ConvertedType);
+            if (decl.IsOutDeclaration())
+            {
+                Assert.Equal(expectedType, typeInfo.ConvertedType);
```

Comment:
It is just an expression with a type, so it can be converted to whatever type is required in its context. The infrastructure for these tests do not support testing for that, and it isn't relevant to the bug being fixed (a crash).